spec/lib/web_of_science/map_names_spec.rb

```diff
@@ -31,8 +31,9 @@
   end
 
   shared_examples 'contains_author_data' do
-    it 'has an authors' do
-      expect(pub_hash[:author]).not_to be_nil
+    it 'has authors with a name key in hash' do
+      expect(pub_hash[:author].size).to be > 0
+      pub_hash[:author].each { |author_name| expect(author_name[:name]).not_to be_nil }
```

Comment:
Looping like this is a bit of an anti-pattern.  Better rspec convention would be:
```ruby
expect(pub_hash[:author]).to all(match a_hash_including(name: be_present))
```

Reply:
fixed and squashed commit